praw/objector.py

```diff
@@ -5,6 +5,16 @@
 class Objector(object):
     """The objector builds :class:`.RedditBase` objects."""
 
+    @staticmethod
+    def fix_dict_keys(d, corrections):
+        """Return a copy of d with keys replaced according to corrections.
+
+        :param d: The dict to be corrected.
+        :param corrections: A dict mapping keys to their replacements.
+
+        """
+        return {corrections.get(k, k): v for k, v in d.items()}
+
```

Comment:
I chose this as an alternative to automatically inferring snake-case from camel-case. That way we can't get bitten by false positives, and it's also possible to make more specific changes. Perhaps this will come in handy with some of the conflicting fields on the `user` dict.